### PR TITLE
Ote web

### PR DESCRIPTION
**Pull request by dmitryagapov**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -7,13 +7,16 @@
 	url = https://github.com/tensorflow/models.git
 [submodule "external/mmdetection"]
 	path = external/mmdetection
-	url = ../mmdetection
+	url = https://github.com/openvinotoolkit/mmdetection
```
Comment (Ilya-Krylov) [inline .gitmodules:10]: please use relative paths

File: docker-compose.cpu.yml [hunk 2]
```diff
@@ -0,0 +1,406 @@
+# Copyright (c) JSC Intel A/O
```
Comment (Ilya-Krylov) [inline docker-compose.cpu.yml:1]: wrong copyright I guess

File: docker-compose.gpu.yml [hunk 3]
```diff
@@ -0,0 +1,402 @@
+# Copyright (c) JSC Intel A/O
```
Comment (Ilya-Krylov) [inline docker-compose.gpu.yml:1]: ditto

File: pytorch_toolkit/ote/ote/__init__.py [hunk 4]
```diff
@@ -1 +1,3 @@
-MODEL_TEMPLATE_FILENAME = 'template.yaml'
+import os
+
+MODEL_TEMPLATE_FILENAME = os.environ.get('MODEL_TEMPLATE_FILENAME', 'template.yaml')
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/ote/ote/__init__.py:3]: what is the purposed of such change?

File: pytorch_toolkit/ote/ote/api.py [hunk 5]
```diff
@@ -48,9 +48,9 @@ def test_args_parser(template_path):
     with open(template_path, 'r') as model_definition:
         config = yaml.safe_load(model_definition)
 
-        parser.add_argument('--test-ann-files', required=True,
+        parser.add_argument('--test-ann-files', required=False,
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/ote/ote/api.py:51]: what is the purpose of such change, is it for custom detector initialization?

Comment (dmitryagapov) [reply pytorch_toolkit/ote/ote/api.py:51]: exactly

File: docker-compose.cpu.yml (hunk 2 above)
Comment (dmitryagapov) [reply docker-compose.cpu.yml:1]: Could you please give me right copyrights?

File: .gitmodules (hunk 1 above)
Comment (dmitryagapov) [reply .gitmodules:10]: Before merge I will revert it

**Comment by Ilya-Krylov** [comment]
`external/mmdetection` should not be changed in this PR, if you are require some features, please let me know

File: docker-compose.cpu.yml (hunk 2 above)
Comment (Ilya-Krylov) [reply docker-compose.cpu.yml:1]: this file has not been updated

**Comment by Ilya-Krylov** [comment]
recreated in https://github.com/openvinotoolkit/training_extensions/pull/377

**Pull request closed without merging**